lib/gps.js

```diff
@@ -432,6 +562,40 @@ GPS.prototype.parseNmeaSentence = function(sentence) {
 
 };
 
+/*
+ * SkyTraq Binary Message Protocol
+ * http://cdn.sparkfun.com/datasheets/Sensors/GPS/Venus/638/doc/AN0003_v1.4.19.pdf
+ */
+GPS.prototype.parseSkyTraqSentence = function(sentence) {
```

Comment:
I guess the same question as above, with same suggestion. Basically: what would happen if this method were called on a GPS that wasn't this model? Probably nothing useful? Maybe an error? 

Maybe GPS class design needs to be revisited? I really wanted this stuff sort out before it was landed, and I thought it had been :\